src/electionguard/serialize.py

```diff
@@ -37,6 +44,50 @@
 )
 
 
+class PaddedDataSize(IntEnum):
+    """Define the sizes for padded data."""
+
+    Bytes_512 = 512 - PAD_INDICATOR_SIZE
```

Comment:
There is only one size now but this will likely change in the future. I could be convinced to modify this.